src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -798,23 +809,15 @@ private function createPackageDefinition($basePath, array $baseUrls, Reference $
             throw new \LogicException('An asset package cannot have base URLs and base paths.');
         }
 
-        if (!$baseUrls) {
-            $package = new ChildDefinition('assets.path_package');
-
-            return $package
-                ->setPublic(false)
-                ->replaceArgument(0, $basePath)
-                ->replaceArgument(1, $version)
-            ;
-        }
-
-        $package = new ChildDefinition('assets.url_package');
+        $package = new ChildDefinition($baseUrls ? 'assets.url_package' : 'assets.path_package');
 
-        return $package
+        $package
```

Comment:
changing this without a need for it makes it harder to merge branches together for no reason